crates/ra_hir/src/expr.rs

```diff
@@ -792,7 +787,33 @@ impl ExprCollector {
             ast::ExprKind::Label(_e) => self.alloc_expr(Expr::Missing, syntax_ptr),
             ast::ExprKind::IndexExpr(_e) => self.alloc_expr(Expr::Missing, syntax_ptr),
             ast::ExprKind::RangeExpr(_e) => self.alloc_expr(Expr::Missing, syntax_ptr),
-            ast::ExprKind::MacroCall(_e) => self.alloc_expr(Expr::Missing, syntax_ptr),
+            ast::ExprKind::MacroCall(e) => {
+                let name = e.name().map(|nr| nr.as_name()).unwrap_or_else(Name::missing);
+
+                if let Some((module, _)) = self.resolver.module() {
+
+                    let def = module.find_macro(&name);
```

Comment:
The find_macro checks in the public_macros field. Does that correspond to all the macros present in the krate or only the ones that are exported.

Reply:
Maybe instead of exposing `module` function on resolver we should add `Resolver::resolve_macro_call`?